src/test/java/uk/gov/ons/ctp/common/event/persistence/VoidEventPersistenceTest.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.ons.ctp.common.event.persistence;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import org.junit.Test;
+
+public class VoidEventPersistenceTest {
+
+  @Test
+  public void testPersistenceSupported() {
+    EventPersistence persistence = new VoidEventPersistence();
+
+    assertFalse(persistence.isFirestorePersistenceSupported());
+  }
+
+  @Test
+  public void testPersistEvent() {
+    EventPersistence persistence = new VoidEventPersistence();
+
+    try {
```

Comment:
just a FYI . since junit 4.13 we can use assertThrows() .   (not to be confused with the similar one George added to the test-framework).  Makes for slightly less wordy way of testing expected exception contents.

Reply:
Interesting. Didn't know it was possible to do that and still check the exception text

Reply:
Have used assertThrows. It reduces code from 7 lines to a more concise 5 lines :)
That is until the autoformatter gets to work and produces a hard to read 11 lines :(